compiler/haskell-ide-core/extension/syntaxes/haskell.tmLanguage

```diff
@@ -0,0 +1,646 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should we not be depending on the normal syntax highlighting one rather than copying it?